src/style.css

```diff
@@ -1,3 +1,92 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+:root {
+  --main-color: #00acdc;
+}
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+}
+
+header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 50px;
+  width: 100%;
+  margin-bottom: 20px;
+}
+
+a.img {
```

Comment:
The selector `a.img` seems incorrect for styling the logo image. In the HTML, the logo image is not wrapped in an anchor tag with a class of 'img'. Consider using `.logo_png` as the selector to match the class used in the HTML.